lhotse/cut/padding.py

```diff
@@ -70,6 +70,10 @@ def has_features(self) -> bool:
     def has_recording(self) -> bool:
         return self.num_samples is not None
 
+    @property
+    def num_channels(self) -> int:
+        return 0
```

Comment:
I think 1 is technically correct, it's all zeros but it's a channel.

```suggestion
        return 1
```